titan/utils.py

```diff
@@ -187,7 +190,9 @@ def override_param(params, param_path, value, delimiter="|"):
     current value to new value
     """
     override_item, last_key = get_param_from_path(params, param_path, delimiter)
-
-    old_val = override_item[last_key]
+    try:
+        old_val = override_item[last_key]
+    except KeyError:
```

Comment:
last_key needs to be changed as well to the int form for line 198 to work properly, right?

Reply:
can you add a blank line between 197 and 198 to appease my OCD?

Reply:
also, is there a test for this?

Reply:
There is in fast-track-msm